les/client_handler.go

```diff
@@ -373,6 +418,31 @@ func (h *clientHandler) removePeer(id string) {
 	h.backend.peers.Unregister(id)
 }
 
+func (h *clientHandler) MinPeerGatewayFee() (map[string]*GatewayFeeInformation, error) {
```

Comment:
Based on the way nature of this method, I would change it to be a method of the cache instead of the `clientHandler`.